RecoLocalTracker/SiStripClusterizer/plugins/clusterGPUv2.cuh

```diff
@@ -0,0 +1,53 @@
+#ifndef _CLUSTER_GPU_KERNEL_v2_
+#define _CLUSTER_GPU_KERNEL_v2_
+
+#include "CUDADataFormats/SiStripCluster/interface/SiStripClustersCUDA.h"
+#include "RecoLocalTracker/SiStripClusterizer/interface/SiStripConditionsGPU.h"
+
+#include <cstdint>
+
+static constexpr int DETS_barrel = 7932;
+static constexpr int DETS_endcap = 7216;
+
+__device__ short indexer3[31214];
```

Comment:
These static allocations definitely won't work with the CMSSW framework, they aren't thread safe and won't work well with multiple devices.  I guess this is all geometry?  If so, leave it and I'll fix it up when I turn it into an ESProducer